tests/models/mamba2/test_modeling_mamba2.py

```diff
@@ -44,6 +44,33 @@
     from transformers.models.mamba2.modeling_mamba2 import Mamba2Cache, Mamba2Mixer
 
 
+class Mamba2ConfigTester(ConfigTester):
+    def _create_config(self, hidden_size: int, num_heads: int, expand: int, head_dim: int):
+        _input_dict = self.inputs_dict.copy()
+        _input_dict["hidden_size"] = hidden_size
+        _input_dict["num_heads"] = num_heads
+        _input_dict["expand"] = expand
+        _input_dict["head_dim"] = head_dim
+        return self.config_class(**_input_dict)
+
+    def test_consistence(self):
```

Comment:
```suggestion
    def test_hidden_size_compatibility(self):
```
Just to have something more descriptive. Should be updated below then in the call :eyes: